### ISSUE TITLE
Button's shadow transition not working in card-actions

### ISSUE THREAD

**Issue by Messier82** [Dogfalo/materialize#2326]
If button(<b>a</b> link with <b>btn</b> class) is placed inside <b>card-action</b> object - button's shadow transition isn't working: on button mouse hover shadow is changing without animation.
```HTML
<div class="card">
    <div class="card-content row" style="color: black;">
        ...Some card content
    <div class="card-action">
        <center><a class="blue lighten-1 btn waves-effect waves-light black-text">Register</a></center>
    </div>
</div>
```

<b>Update:</b>
In Chrome dev-tool noticed, that card-action class is overwriting btn's transition rule with own rule.

**Comment by acburst** [comment]
Fixed in 01014ed. Thanks for pointing this out!

**Comment by Messier82** [comment]
Thank you for fast fix!

**Comment by Messier82** [comment]
@acburst Unfortunately, after your fix tabs started to work very strange - there appeared vertical scrollbar(in <b>tabs</b> object)

**Comment by acburst** [comment]
Not sure what's happening with tabs. It shouldn't be affected. I did however make an embarrassing mistake in my last commit >.<

Fixed in 520b367. 

If you still have problem with tabs, do you mind posting some code so I can replicate?

**Comment by m00k1e** [comment]
Hello!

 

New message, please read <http://suntech7.com/transport.php?mn8y>

 

hello@mookiecreative.com
